### ISSUE TITLE
Schematic: Red Firework | Can't learn it as Alliance player

### ISSUE THREAD

**Issue by rockerlalee** [dalaranwow/dalaran-wow#5297]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**:
I purchased the Schematic: Red Firework (http://www.wowhead.com/item=18647/schematic-red-firework) from the Neutral Auction House for my Alliance character (Kommando). Despite having the necessary skill in Engineering to be able to learn this schematic, I can't. It says "You can never learn this recipe".

**Expected behaviour**: Said schematic is not a Horde-only one, thus I should be able to use it properly. 

**Steps to reproduce the problem**:

1. It has been reported many times in other servers, that upon successfully learning said schematic, it teaches Blue Firework (http://www.wowhead.com/item=18649/schematic-blue-firework) for Alliance players instead - and vice versa for Horde Players.

2. Some players said that via GM assistance they've received another schematic which they've been able to learn.

**Include proofs for this behaviour**
http://www.wowhead.com/item=18647/schematic-red-firework#comments
"By Tricksy (766 – 9·6) on 2010/11/23 (Patch 4.0.3) |   |  
I purchased this schematic on the Alliance AH, however when I opened the mail containing it, it turned into "Schematic: Blue Firework". Since I already knew that, I couldn't learn it.

I petitioned a GM about it and the next day I had a response from Blizzard in the mail saying that they had replaced my Schematic with the red one. However, it was still the Blue one. 

I figured that whatever happened the first time had happened again so I responded to the mail and within 5 mins I got a GM whispering me, he sent me a new Red Firework recepie in the mail, and once again it turned blue when opened. After this the GM asked me to log onto another char so that he could fix it. When I logged back in he had simply added the recepie to my "known" repecies."

ID for Schematic: Red Firework: 18647

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/4413
